src/lib/viewers/BaseViewer.js

```diff
@@ -1090,7 +1112,7 @@ class BaseViewer extends EventEmitter {
         const { showAnnotationsControls, file } = this.options;
         const { permissions, extension } = file || {};
 
-        if (!this.hasAnnotationPermissions(permissions)) {
+        if (!this.hasAnnotationCreatePermission(permissions) && !this.hasAnnotationViewPermission(permissions)) {
```

Comment:
These changes will ensure that the controls won't be shown until the backend enables support for the file type being previewed.